### DISCUSSION TITLE
Multi-GPU Utilization

### DISCUSSION THREAD

**Discussion by RyzenAdvanced** [Blinue/Magpie#152] [Ideas]
Dear Developer,

For instance in a gaming laptop such as:
ASUS - ROG Strix G15 Advantage Edition

There is built-in VEGA 8 GPU and Dedicated 6800M GPU.

Perhaps, could allow the Vega to assist in upscaling and offload from the dedicated GPU.

Just as idea :)

Thanks!

**Comment by Blinue** [comment]
#88

**Comment by RyzenAdvanced** [comment]
@Blinue since these asus laptop asus model has Vega 8 GPU as iGPU these feature could work, right ?

Maybe within Magpie application itself, you can make GPU selectot:
Main GPU
Second GPU 

And allow user to set, which GPU does the upscaling.

Just as idea :)
